src/pages/index.tsx

```diff
@@ -100,11 +99,20 @@ const Index: React.FC<IndexProps> = ({ fetchedData }) => {
 export default Index;
 
 export const getServerSideProps: GetServerSideProps = async () => {
-  const fetchedData = await getIdentities();
+  const sortedIdentities = await getIdentities().then((result) => {
+    if (result instanceof Error) {
+      throw new Error();
+    }
+    return sortIdentities(result);
+  });
+
+  if (Object.values(sortedIdentities).length === 0) {
+    throw new Error("No identities found");
+  }
```

Comment:
@Fenntasy I'd like your thoughts on this one. This is the quick solution we found to handle the case where the identities field received in `sortIdentities` is null or undefined, and returns an empty object.

Reply:
Do we need to throw errors though? 

Couldn't we return `{ phoneIdentities: [], emailIdentities: [] }` and just display "No emails" and "No phones" when the array is empty in the UI?

Reply:
OK fixed 1217fa4